cdap-data-fabric/src/main/java/co/cask/cdap/store/StoreDefinition.java

```diff
@@ -75,6 +78,39 @@ public static void createTables(StructuredTableAdmin tableAdmin) throws IOExcept
     }
   }
 
+  /**
+   * Schema for workflow table
+   */
+  public static final class WorkflowStore {
+    public static final StructuredTableId WORKFLOWS = new StructuredTableId("workflows");
```

Comment:
The previous code called this workflow statistics, I'd recommend using the same name here